pytorch_lightning/lite/lite.py

```diff
@@ -399,17 +399,16 @@ def seed_everything(seed: Optional[int] = None, workers: Optional[bool] = None)
         return seed_everything(seed=seed, workers=workers)
 
     def _run_impl(self, run_method: Callable, *args: Any, **kwargs: Any) -> Any:
-        self._strategy.setup_environment()
-
```

Comment:
Why is this gone?

Reply:
moved to L411. Tests would have failed if I'd have removed it.